src/Services/Azure/Kusto/KustoService.cs

```diff
@@ -118,27 +107,15 @@ public async Task<List<string>> ListDatabases(
     {
         ArgumentException.ThrowIfNullOrEmpty(clusterUri, nameof(clusterUri));
 
-        var kcsb = await CreateKustoConnectionStringBuilder(
```

Comment:
nit; for argument validations, we could use the `ValidateRequiredParameters` from the `BaseAzureService` this service class inherits from